docs/developer-guide/create-a-backend-add-on.md

```diff
@@ -0,0 +1,21 @@
+---
+myst:
+  html_meta:
+    "description": "How to create a backend add-on"
+    "property=og:description": "How to create a backend add-on"
+    "property=og:title": "Create a backend add-on"
+    "keywords": "Plone 6, backend, add-on"
+---
+
+(create-a-backend-add-on-label)=
+
+# Create a backend add-on
+
+This section explains how a developer can create an {term}`add-on` for the Plone backend.
+You will use a framework called {term}`GenericSetup` and a tool called {term}`Buildout`.
```

Comment:
I don't quite get your vision for this section yet.

Most of the documentation that can be written about how to create a backend add-on will be relevant regardless of how Plone was installed, and also regardless of whether the add-on was created from a Cookieplone template, bobtemplates.plone, or by hand.

So mentioning buildout here is off topic.

And learning about `GenericSetup` is a piece of what goes into creating an add-on, but it's certainly not the only or the most important piece.

Reply:
I created this file to jump start something for @petschki to work on. I have no vision about this whatsoever. If the snippets aren't needed, they can be tossed out.

I moved the snippets here from another document that had them as headings in a how-to guide, instead of in a conceptual guide. IOW no one does "How to Buildout and GenericSetup".